libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -240,6 +241,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
     private val internalFallbackVersionsObserver: FallbackVersionsObserver
     private val routeAlternativesController: RouteAlternativesController
     private val routeRefreshController: RouteRefreshController
+    private var routeRefreshJob: Job? = null
```

Comment:
I think it would make sense to create something like `routeScope` and start all the route related tasks from it, so that when we update route we don't need to cancel all individual jobs - just recreate the scope. Maybe next time 🙂

Reply:
could you cut a ticket

Reply:
thanks. It was easier to implement than to cut a ticket 😆